reactive/filebeat.py

```diff
@@ -32,6 +38,29 @@ def render_filebeat_template():
         status_set('active', 'Filebeat ready.')
 
 
+@restart_on_change({
+    LOGSTASH_SSL_CERT: ['filebeat'],
+    LOGSTASH_SSL_KEY: ['filebeat'],
+    })
+@when('beat.render')
+@when('apt.installed.filebeat')
```

Comment:
These `@when` decorators need to go above the `@restart_on_change`. As it is now, the `@when` decorators will have charms.reactive calling the unwrapped `render_filebeat_logstash_ssl_cert`, not the wrapped version created by `@restart_on_change`.